docs/install.md

```diff
@@ -0,0 +1,262 @@
+# Install and Serve
+
+## Contents of this Document
+
+* [Install TorchServe](#install-torchserve)
+* [Serve a Model](#serve-a-model)
+* [Quick start with docker](#quick-start-with-docker)
+* [Contributing](#contributing)
+
+## Install TorchServe
+
+Conda instructions are provided in more detail, but you may also use `pip` and `virtualenv` if that is your preference.
+**Note:** Java 11 is required. Instructions for installing Java 11 for Ubuntu or macOS are provided in the [Install with Conda](#install-with-conda) section.
+
+### Install with pip
+To use `pip` to install TorchServe and the model archiver:
+
+```
+pip install torch torchtext torchvision sentencepiece
+pip install torchserve torch-model-archiver
+```
+
+### Install with Conda
+_Ubuntu_
+
+1. Install Java 11
+    ```bash
+    sudo apt-get install openjdk-11-jdk
+    ```
+1. Install Conda (https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html)
+1. Create an environment and install torchserve and torch-model-archiver
+    For CPU
+    ```bash
+    conda create --name torchserve torchserve torch-model-archiver pytorch torchtext torchvision -c pytorch -c powerai
```

Comment:
Will need to grab the latest from #286.

Reply:
Pulled in latest install changes from PR#286